biz.aQute.bndlib/src/aQute/bnd/help/Syntax.java

```diff
@@ -705,6 +705,8 @@
 		new Syntax(TESTCONTINUOUS,
 			"Do not exit after running the test suites but keep watching the bundles and rerun the test cases if the bundle is updated.",
 			TESTCONTINUOUS + "=true", "true,false", Verifier.TRUEORFALSEPATTERN),
+		new Syntax(TESTTERMINATE, "Do not terminate the remote framework after running the test suites.",
```

Comment:
I guess this description should be "Terminate the remote" rather than "Do not terminate", right?

Reply:
fixed